slsim/Deflectors/velocity_dispersion.py

```diff
@@ -228,10 +226,10 @@ def vel_disp_sdss(sky_area, redshift, vd_min, vd_max, cosmology, noise=True):
     .. [1] Choi, Park and Vogeley, (2007), astro-ph/0611607, doi:10.1086/511060
     """
     # SDSS velocity dispersion function for galaxies brighter than Mr >= -16.8
-    phi_star = 8.0 * 10 ** (-3) / cosmology.h**3
-    vd_star = 161
-    alpha = 2.32
-    beta = 2.67
+    phi_star = 2.099e-2 * cosmology.h**3
```

Comment:
just checking, are these the numbers on Choi et al. 2007 or are these from a different paper?

Reply:
Ohh, these numbers are from Bernardi et al. 2010. I updated the reference.